docs/checklist_applicationclass_2-markdown_v1.1.md

```diff
@@ -0,0 +1,157 @@
+﻿# Checklist for Application Class 2
+> The checklist is in version 1.1. It refers to the DLR Software Engineering Guidelines [Reference Manual](https://doi.org/10.5281/zenodo.1344612) in version 1.0.0.
+
+<!--
+## Usage Hints
+This checklist provides recommendations for software development. It is primarily intended for software developers for the self-assessment of developed software and as a source of ideas for further development. The checklist does not provide any new, revolutionary approaches to software development. However, it helps to not forget necessary, essential steps of software development. In addition, the recommendations can serve as an argumentation aid. 
+
+The recommendations are created with a focus on know-how maintenance and good software engineering practice. They help to maintain the sustainability of the developed software. The recommendations encourage the use of tools, the creation of documentation, the establishment of processes and adherence to principles. When assessing a recommendation, it is recommended to consider to what extent the aspect mentioned is implemented and whether there is a need for improvement. This could be implemented as follows: 
+
+* Is there currently no need for improvement and is the recommendation addressed appropriately? Status: **ok** 
+* Is there any potential for improvement that should be implemented in the near future? Status: **todo**, record the need for improvement 
+* Is the recommendation not yet relevant but could be helpful in a later development phase? Status: **future** 
+* Is the recommendation not meaningfully implementable within the development context? Status: **n.a.** (not applicable) explain the reason 
+
+In case of questions, you can contact the Software Engineering Contact of your institute or facility.
+
+> Please note the status between "[]" and list remarks below a recommendation.
+
+-->
+
+## Summary of Results
+The software CosmoScout VR implements 42 recommendations of application class 2. 
+
+The focus of future improvements is on improving the testing capabilities.
```

Comment:
We clearly need to improve our testing. But I also think we should document the high level requirements such as real-time, multi-display etc. We should also document out main partners in DLR.

Reply:
```suggestion
The focus of future improvements is on improving the testing capabilities and working out a requirements documentation.
```